### ISSUE TITLE
qlik sense september 2020

### ISSUE THREAD

**Issue by saso1970** [jbellizzi/qlik-dev-suite#1]
Extension can not be add in qlik sense September 2020

**Comment by jbellizzi** [comment]
Thanks for the tip! I am currently setting up a September 2020 instance and will be testing on there shortly

**Comment by jbellizzi** [comment]
@saso1970 I've been able to confirm it is working on my end in September 2020. Can you specify if you're running on a specific version (i.e. September 2020 Patch 1)

If you could provide any other details on how you are trying to add the extension and what failure you're getting that would be helpful

**Comment by saso1970** [comment]
Yes I use September sr1 .No error when I  drop the extension is nothing
happened  no extension at all on sheet.

**Comment by jbellizzi** [comment]
When you drop the extension onto the page, it should move itself up to the top of the sheet next to the sheet name, so if the sheet is extended to have vertical scroll, you may need to scroll up to see it. You should also make sure you are viewing the sheet in edit mode. The extension hides itself when you leave edit mode so you won't be able to see it.

if you still are having an issue, can you check the javascript console to see if there are any errors occurring? (on chrome you can do this by hitting f12 in your browser, or going to View -> Developer -> JavaScript Console)

**Comment by saso1970** [comment]
I Upgrade on September SR2  same Problem Extension  can not be add on sheet and in console i have only those Errors Sent from Mail for Windows 10 From: John BellizziSent: Saturday, October 10, 2020 12:15 AMTo: jbellizzi/qlik-dev-suiteCc: saso1970; MentionSubject: Re: [jbellizzi/qlik-dev-suite] qlik sense september 2020 (#1) When you drop the extension onto the page, it should move itself up to the top of the sheet next to the sheet name, so if the sheet is extended to have vertical scroll, you may need to scroll up to see it. You should also make sure you are viewing the sheet in edit mode. The extension hides itself when you leave edit mode so you won't be able to see it.if you still are having an issue, can you check the javascript console to see if there are any errors occurring? (on chrome you can do this by hitting f12 in your browser, or going to View -> Developer -> JavaScript Console)—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.
